src/components/calculator/calculator.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+import './calculator.css';
+
```

Comment:
- [ ] OPTIONAL
 Following best practice, this file `calculator.js` should begin with an upper case, like: `Calculator.js`. Kindly endeavor to improve upon the same.

Reply:
- Thanks @rloterh 🙏 for reviewing my pull request I appreciate your review  I will modify👌 this to enhance my code thanks for your hints to make me a professional I am very happy to hear this optional suggestion I will work on it 💯 
### Thanks again for reviewing my PR  and have a good day😊